gulpfile.js

```diff
@@ -35,11 +35,11 @@ gulp.task("6to5", function () {
 });
 
 gulp.task("sass", function () {
-  return gulp.src("server/src/**/*.scss").pipe(sass()).pipe(gulp.dest("server/dist/css"));
+  return gulp.src("server/src/**/*.scss").pipe(sass()).pipe(gulp.dest("server/dist/static/css"));
```

Comment:
Yeah, src/ is becoming less necessary – adding static/ made the routing simpler and clearer.